### ISSUE TITLE
Server rolling back chunks instead of saving them.

### ISSUE THREAD

**Issue by VermillionX** [GTNewHorizons/NewHorizons#1094]
I left my house to go outside during a blood moon. During my circuit of my town, the server decided to roll back half of my base an hour. Deleting everything i'd done in that hour.
My resource storage wasn't rolled back, as it's in the adjacent chunk. So I lost all the resources used to craft everything that was on the other side of my base. So what did the server decide to just destroy for no reason?
Half of a Tier 2 rocket's components, a full Nuclear reactor + 16 quad thorium fuel rods (the second time it's done this, I suspect. Since this is the second time i've had to make my reactor, which also spontaneously vanished the same way), 1 Ender IO Fluid Tank, 2 stacks of titanium, 3 stacks of steel and 1 stack of lead.
Not cool.

**Comment by Dream-Master** [comment]
What you mean with roll back? We had in the past that some chunks be reseted for no reason and be stone (underground base) and all stuff in this chunk are bee gone. If this happens and you play on the server just ask someone to refund all your stuff back. Sometimes chunks becomes corrupt for no reason.

**Comment by VermillionX** [comment]
I made a nuclear reactor + 6 chambers, EBF'd a bunch of things, made 16 Quad-Thorium Fuel Rods and set my HV assembler to make a pile of Tier 2 Heavy Duty ingots, melted some stuff and a blood moon happened so I went to run around outside at night and kill lots of mobs with my Epic-MLP-MLG-uberpro-wtfbbq-machinegun-Crossbow.
When I returned to my base, everything i'd done in the past hour in only one chunk had rolled back. Sulfur, Sticks and torches now occupy my HV assembler, instead of the T2HD ingots, so are the components they were made from. By EBF has 32 Cobalt in it instead of a stack of Hot Titanium ingots. The reactor, chambers and thorium rods also went "poof", but because my drawers that contain the thorium, lead, titanium, aluminium, steel, platinum and rubber used to make those things are all in a seperate chunk than the one my assembler and EBF are in they did not roll back.
Basically, the server deleted an hours worth of work, and didn't even refund me the resources to repeat it, and evidently, this isn't the first time it's decided to just not save my progress.

**Comment by Namikon** [comment]
Do you remember if the backup process was active while this all happened?

**Comment by VermillionX** [comment]
Not sure how you'd check, though 15 minutes later there was the usual backup-lag message, followed by the backup complete announcement. So i'd guess it works.

**Comment by Namikon** [comment]
It is unlikely to be the cause, but as saving to disk is stopped while the backup process is running, (Changes are kept in memory and get written to disk once the backup is completed) it *could've* been the reason for a chunk reset. But It bugs me somehow that you wrote 

>  Since this is the second time i've had to make my reactor, which also spontaneously vanished the same way

Now I'm daring to assume that something might be wrong with some mod you're using in that chunk. But to proof that theory (Or proof that it's wrong) it would be required that you or someone else is able to reproduce this a third time...

**Comment by VermillionX** [comment]
I repeated the same steps I did last time and sure enough it rolled back again. This time I stuck paper in several slots my cells/fluid chest and ran a lap of town, killed some mobs. Came back and sure enough, the paper is gone.
![2017-02-08_23 46 32](https://cloud.githubusercontent.com/assets/10645298/22734113/74112c5a-ee59-11e6-89f2-dd2d3b7f990d.png)
![2017-02-08_23 48 22](https://cloud.githubusercontent.com/assets/10645298/22734117/7adab088-ee59-11e6-8d2a-7a87abcc0604.png)
